lib/partitions.js

```diff
@@ -130,6 +131,19 @@ Partition.prototype.produce = function() {
     }.bind(this));
 }
 
+/**
+ * Get a Partitions resource representing the partitions for this topic. This does not request any metadata since you may
+ * use the Partitions resource to get a specific partition; call list() on the resulting Partitions resource to get a
+ * full listing with metadata.
+ *
+ * @param res function(err,
+ */
+Partition.prototype.__defineGetter__("messages", function() {
+    if (!this._messages)
+        this._messages = new PartitionMessages(this);
+    return this._messages.messages.bind(this._messages);
```

Comment:
I didn't immediately get how to use this -- partly because there wasn't an example, but more so because unlike other cases where a subresource returns an object which we then call actions on (e.g. subresource `Partition` has action method `produce`), this `messages` subresource is just a method that can be invoked.

I don't necessarily think this is a problem, especially if we're convinced there will be no other action than the single `GET` HTTP verb on this subresource. However, I'd like to a) verify that's likely to be the case and b) make sure we justify this deviation from the usual pattern if we decide to make this an exception. Thoughts?